db/migrate/20190224103931_add_git_puppet_settings.rb

```diff
@@ -2,8 +2,8 @@
 
 class AddGitPuppetSettings < ActiveRecord::Migration[5.2]
   def change
-    add_column :app_settings, :puppet_init_path, :string, default: '/'
```

Comment:
Why was an existing migration edited instead of creating a new one, which removes the defaults?

The migration will not be applied on a system that has already applied the unchanged `db/migrate/20190224103931_add_git_puppet_settings.rb` migration, e.g. the dev deployment.
Changing migrations requires everyone to drop and recreate the db, working against the idea of migrations.

Reply:
I just put the default in the seeds.rb and there is no need for changes in the already existing database (I do not want a new value, I just wanted to move where it is initialized) and in the future it should be initialized with the seed if it is created new.

Reply:
> there is no need for changes in the already existing database
Good point.